content/rule_book/scoring.md

```diff
@@ -0,0 +1,299 @@
+# Scoring
+
+## Scoring Individual and Team Points
+
+### Individual Points
+
+All points that occur during a bonus question or during overtime do not contribute to the individual score of a quizzer.
+
+#### Points Earned
+
+- +20 points for every correct question and toss-up question
+- +10 points for every quiz-out without error
+
+#### Points Deducted
+
+- -10 points for 2nd and subsequent personal errors
+- -10 points for 3rd personal foul
+
+### Team Points
+
+All points earned or deducted by an individual are to be counted towards the team's points.
+
+#### Points Earned
+
+- +10 points for the 1st correct answer given by the 3rd and subsequent quizzer on the team
+- +20 points for every correct bonus question before question number 17
+     - In 2-team quizzes, this scoring rule does not apply
+- +10 points for every correct bonus question after and including question number 17
+     - In 2-team quizzes, all correct bonus questions are worth 10 points
+
+#### Readiness Bonus
+
++20 points will be awarded to each team present at the scheduled start time of the quiz.
+
+Teams that arrive late due to quizzing in another room are excused from the forfeiture of these points. If a single quizzer is late, the coach can decide to keep the +20 points by keeping the late quizzer out the whole quiz, or forfeit the points by subbing the quizzer in after question number 1. This decision must occur before the quiz has been started.
+
+##### Logic
+
+    Set the readiness bonus to 20.
+
+### Points Deducted
+
+Team errors are the sum of all individual, non-bonus errors.
+
+- -10 points for every team error starting at team error number 3
+- -10 points for every error on a question or toss-up question starting at question number 17
```

Comment:
This is specific to 20-question quizzes. We can disambiguate by saying "-10 points for every error once error points have begun"

Reply:
Though I guess this states the defaults, which are overridden by the specific quiz types.  Carry on.

Reply:
:)